src/diagnostics/AdvancedCameraTest.ts

```diff
@@ -26,7 +39,11 @@ export default class AdvancedCameraTest extends TestSuite {
     this.addTest(new CameraResolutionTest([[3840, 2160]], options));
     this.addTest(new CameraResolutionTest([[4096, 2160]], options));
   }
-  start () {
+
+  /**
+   * Start the test
+   */
+  public start (): Promise<any> {
```

Comment:
I wonder if we could make this type more prescriptive. Results should all look similar, I imagine.

Reply:
Ahh, you are right. I am already passing back a typed object. The type doesn't have an interface, but we know what it is. I will make sure to change this before my final PR.

Reply:
I updated this return type in `AdvandedCameraTest`. I see what you are saying, though. I misunderstood your initial comment. I added a ticket in to standardize the return types across the tests. Right now, they return fairly different results per test. I definitely agree that they should all return a similar type.